### DISCUSSION TITLE
RoderickShao_notebook/%E9%80%9A%E8%AF%86%E5%9F%BA%E7%A1%80%E8%AF%BE/%E6%A6%82%E7%8E%87%E8%AE%BA%E4%B8%8E%E6%95%B0%E7%90%86%E7%BB%9F%E8%AE%A1/%E7%AC%AC%E5%85%AD%E7%AB%A0%20%E7%BB%9F%E8%AE%A1%E9%87%8F%E4%B8%8E%E6%8A%BD%E6%A0%B7%E5%88%86%E5%B8%83/

### DISCUSSION THREAD

**Discussion by giscus** [RoderickShao/RoderickShao_notebook#14] [Announcements]
# RoderickShao_notebook/%E9%80%9A%E8%AF%86%E5%9F%BA%E7%A1%80%E8%AF%BE/%E6%A6%82%E7%8E%87%E8%AE%BA%E4%B8%8E%E6%95%B0%E7%90%86%E7%BB%9F%E8%AE%A1/%E7%AC%AC%E5%85%AD%E7%AB%A0%20%E7%BB%9F%E8%AE%A1%E9%87%8F%E4%B8%8E%E6%8A%BD%E6%A0%B7%E5%88%86%E5%B8%83/

RoderickShao's NoteBook

https://roderickshao.github.io/RoderickShao_notebook/%E9%80%9A%E8%AF%86%E5%9F%BA%E7%A1%80%E8%AF%BE/%E6%A6%82%E7%8E%87%E8%AE%BA%E4%B8%8E%E6%95%B0%E7%90%86%E7%BB%9F%E8%AE%A1/%E7%AC%AC%E5%85%AD%E7%AB%A0%20%E7%BB%9F%E8%AE%A1%E9%87%8F%E4%B8%8E%E6%8A%BD%E6%A0%B7%E5%88%86%E5%B8%83/

**Comment by Provence-24** [comment]
第六章内容尝试梳理：本章中开始研究样本，聚焦样本本身的特性，因此我们有了统计量来描述一定的特征，如：样本均值（简称均值），样本标准差（简称标准差）。我们按照第五章的思路，希望确定这些统计量的分布。因此我们引入三种标准的分布，对于这三种标准分布需要掌握定义式、均值、方差。至于样本的研究做什么用，这一章并没有涉及。当然现在我对于三种标准的分布还不甚熟悉，对于本章的本质把握仍存在不少偏差，做一些题后，再来发表感想。
